src/ad-hoc-visualizations/needs-review/visualization.tsx

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { AdHocTestkeys } from 'common/configs/adhoc-test-keys';
+import { TestMode } from 'common/configs/test-mode';
+import { VisualizationConfiguration } from 'common/configs/visualization-configuration';
+import { FeatureFlags } from 'common/feature-flags';
+import { Messages } from 'common/messages';
+import { VisualizationType } from 'common/types/visualization-type';
+import { generateUID } from 'common/uid-generator';
+import { AdhocIssuesTestView } from 'DetailsView/components/adhoc-issues-test-view';
+import { VisualizationInstanceProcessor } from 'injected/visualization-instance-processor';
+import * as React from 'react';
+
+export const NeedsReviewAdHocVisualization: VisualizationConfiguration = {
+    key: AdHocTestkeys.NeedsReview,
+    testMode: TestMode.Adhoc,
+    getTestView: props => <AdhocIssuesTestView {...props} />,
+    getStoreData: data => data.adhoc.needsReview,
+    enableTest: data => (data.enabled = true),
+    disableTest: data => (data.enabled = false),
+    getTestStatus: data => data.enabled,
+    displayableData: {
+        title: 'Needs Review',
```

Comment:
We use sentence casing for our titles, so this would be: 'Needs review'

Reply:
changed